jsapp/js/components/processing/sidebar/sidebarDisplaySettings.module.scss

```diff
@@ -0,0 +1,46 @@
+@use '~kobo-common/src/styles/colors';
+@use 'scss/sizes';
+
+.root {
+  display: flex;
+  justify-content: right;
+  padding-top: sizes.$x18;
+  padding-bottom: sizes.$x10;
+}
+
+.description {
+  padding-bottom: sizes.$x30;
+}
+
+.display {
+  padding-bottom: sizes.$x40;
```

Comment:
Last display shouldn't get this padding, you can just add `:not(:last-child)` to the selector